cmd/meroxa/turbine/ruby/cli.go

```diff
@@ -17,26 +17,27 @@ type turbineServer interface {
 	GracefulStop()
 }
 
-type recordClient struct {
+type specBuilderClient struct {
 	*grpc.ClientConn
 	pb.TurbineServiceClient
 }
 
 type turbineRbCLI struct {
 	logger            log.Logger
 	appPath           string
-	runServer         turbineServer
-	recordServer      turbineServer
-	recordClient      recordClient
 	grpcListenAddress string
+
+	bs turbineServer
+	bc specBuilderClient
+	rs turbineServer
```

Comment:
I'm confused what `bs` and `rs` stands for 🙃

Reply:
Never mind, I see below that it's build and record servers... I think this could benefit from being more explicit

Reply:
Ugh yea, I was pretty terse here with the naming. What went in my head was that these are not exported variables and then the initializer defines them so anyone looking at the code should kind of know what they are based on the initialisation. 

I can rename them it that’s your preference

Reply:
I was going to go with builder and runner :)